src/obfl-specification.html

```diff
@@ -2777,56 +2775,10 @@ <h3 id="L2762">mark-capital-letters</h3>
 preference for marking capital letters should be applied.</p>
 
 <p>The value of the <code>mark-capital-letters</code> attribute can either be 'true' or 'false'.
-The value 'true' indicates that a text algorithm add capital markings to the
-text contents. The value 'false' indicates that a text algorithm should
+The value 'true' indicates that we add capital markings to the
+text contents. The value 'false' indicates we should
```

Comment:
The "we"-form isn't used anywhere in the specification. Sorry, I think it needs to stay a bit uniform.

Who is "we" anyway? The info in this attribute should go straight to the braille translator (this is how it's currently done in Dotify and I think it's the right way). So I'd replace "we" with "the braille translator". The issue however is that OBFL doesn't have the notion of a braille translator. At least it is not explained anywhere what this term means and how it interacts with everything. This may have been intentional because in the very first paragraph it says "[OBFL] is designed for braille, but can also be used in other fixed character width contexts".